lib/circlex/emulator/router.ex

```diff
@@ -37,6 +37,7 @@ defmodule Circlex.Emulator.Router do
 
   # Payments API
   forward("/v1/payments", to: Circlex.Emulator.Api.Payments.PaymentsApi)
+  forward("/v1/balances", to: Circlex.Emulator.Api.Core.BalancesApi)
```

Comment:
Is this a different, unrelated fix? If so, we should make it a different PR.

Reply:
this fixes the following error in treasury's circle_admin page:

![](https://cdn.discordapp.com/attachments/1006245024587067433/1011324890265170031/Screen_Shot_2022-08-22_at_1.23.32_PM.png)
will make a separate PR